ui-framework-documentation.md

```diff
@@ -459,6 +459,39 @@ sub edit_proposal {
 
  edit_proposal({is_lvm => 1, has_separate_home => 1});
 ``` 
+
+### Abbreviated prefixes for libuyi widgets
```

Comment:
technically they are YuiRestClient Widgets (YuiRestClient::Widget) it is our construction to map libyui Widgets developed by YaST developers, libyui widget is for example: YRichText which lives here: https://github.com/libyui/libyui